newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -134,7 +135,8 @@ class Instruction extends React.Component<Props, *> {
             >
               {ParameterRenderingService.renderParameterString(
                 parameterType,
-                formattedTexts.getString(i)
+                formattedTexts.getString(i),
+                this.props.renderObjectThumbnail
```

Comment:
Sounds like a good idea to pass renderObjectThumbnail 👍

Reply:
Thank you :) I thought it will create less work than passing layout and project